### PR TITLE
#166790014 Users Should Be Able to Like a Specific Comment

### PR DESCRIPTION
**Pull request by niyoceles**
#### What does this PR do?
Enables a user to like a comment
#### Description of Task to be completed
- To set up like a comment functionality
#### How should this be manually tested?
1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-user-like-specific-comment-166790014
npm install
npm start
```
2. Create a .env file and add the following snippets
```
DATABASE_URL=postgres://username:password@localhost:5432/authorshaven
DATABASE_=postgres://username:password@localhost:5432/authorshaven_test
```
3. Install `sequelize` globally
```
sudo npm install -g sequelize
```
4.  Create a database using `sequelize db:create`
5. Run the following requests in Postman using `localhost:3000`
|  `POST`  |  `/api/article/comment/:articleId/like`  |  `like a comment`  |
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166790014](https://www.pivotaltracker.com/story/show/166790014)
#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

File: tests/likeComment.test.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        expect(body).to.have.property('user');
+        expect(body.user).to.have.property('email');
+        expect(body.user).to.have.property('username');
+        expect(body.message).to.equals('logged in');
+        otherUserToken.push(body.user.token);
+        done();
+      });
+  });
+
+  it('should login an user', (done) => {
```
Comment (kodek-sleuth) [inline tests/likeComment.test.js]: You only need one test block of logging in a user

**Review by kodek-sleuth — CHANGES_REQUESTED** [review]

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by kodek-sleuth — DISMISSED** [review]

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by IsaiahRn — DISMISSED** [review]

**Review by elemanhillary-zz — APPROVED** [review]
ready for merge

**Review by IsaiahRn — APPROVED** [review]

**Review by dmithamo — APPROVED** [review]

**Pull request merged**